test/e2e/replacement_values.py

```diff
@@ -175,4 +221,6 @@
     "SAGEMAKER_EXECUTION_ROLE_ARN": get_bootstrap_resources().ExecutionRoleARN,
     "MODEL_MONITOR_ANALYZER_IMAGE_URI": f"{MODEL_MONITOR_IMAGE_URIS[get_region()]}/sagemaker-model-monitor-analyzer",
     "CLARIFY_IMAGE_URI": f"{CLARIFY_IMAGE_URIS[get_region()]}/sagemaker-clarify-processing:1.0",
+    "ENDPOINT_INSTANCE_TYPE": ENDPOINT_INSTANCE_TYPES[get_region()],
```

Comment:
nit: since most regions, we can use same instance type, can we use
`ENDPOINT_INSTANCE_TYPES.get(get_region(), ml.c5.large)`

Reply:
Makes sense, added changes below